tutorials/ChannelProvider.publish.md

```diff
@@ -0,0 +1,12 @@
+Publish an action and payload to every connected client.
+
+### Example
+```js
+(async ()=> {
+    const provider = await fin.InterApplicationBus.Channel.create('channelName');
+
+    provider.onConnection(async (identity, payload) => {
+        await provider.publish('new-connection', identity);
+    });
```

Comment:
Publish synchronously returns an array of promises.  We should highlight that here somewhere. Maybe await Promise.all instead of await?  Should probably mention it above as well since it doesn't really conform to the other methods that all return promises.

Reply:
good point.